src/network/networkprotocol.h

```diff
@@ -644,7 +645,13 @@ enum ToClientCommand
 		std::string bytes_B
 	*/
 
-	TOCLIENT_NUM_MSG_TYPES = 0x61,
+	TOCLIENT_FORMSPEC_PREPEND = 0x61,
+	/*
+		u32 len
```

Comment:
not that it matters but:
a formspec would comfortably fit into a maximum of 64k chars

Reply:
TOCLIENT_SHOW_FORMSPEC uses u32, but it's probably better for this not to

Reply:
`NetworkPacket& NetworkPacket::operator>>(std::string& dst)` is responsible for reading "short" strings